src/app/components/errors/server-error/server-error.component.html

```diff
@@ -0,0 +1,3 @@
+
+<img src="../../../../assets/images/500-Internal-Error.gif">
```

Comment:
Should we use our own created GIF image here or downloaded image is fine as well?

Reply:
The created GIF will be good but as long as not much customization is required we can use downloaded images as well.

Reply:
@RishabhJain2018 , do EvalAI AngularJS uses own created images or downloaded one? Should we use EvalAI images/SVG only ? @lunayach